src/main/scala/eventstore/operations/Operation.scala

```diff
@@ -45,6 +46,12 @@ private[eventstore] object Operation {
       case Pong                 => None
       case HeartbeatRequest     => Some(simple(HeartbeatRequestInspection))
       case HeartbeatResponse    => None
+      case x: Ps.Connect        => Some(retryable(PersistentSubscriptionOperation(x, pack, client, connected)))
+      case x: Ps.Create         => Some(simple(CreatePersistentSubscriptionInspection(x)))
+      case x: Ps.Update         => Some(simple(UpdatePersistentSubscriptionInspection(x)))
+      case x: Ps.Delete         => Some(simple(DeletePersistentSubscriptionInspection(x)))
+      case x: Ps.Ack            => None
+      case x: Ps.Nak            => ???
```

Comment:
I can't think of when a Nak would be sent. If you could tell me where and when it is, then I'll happily implement it and put the corresponding code that Naks in the actor.